### PR TITLE
added transformations

### PR DESCRIPTION
**Pull request by malogal**
transformations with push physics are added
teleportation does not work

### CODE REVIEW

File: scenes/characters/Player.gd [hunk 1]
```diff
@@ -243,6 +243,13 @@ func _physics_process(delta):
 	if not hasVisitedForest:
 		if position.y < -390:
 			hasVisitedForest = true
+	
+	#pushing rigidbodies
+	#move_and_slide()
+	for i in get_slide_collision_count():
+		var c = get_slide_collision(i)
+		if c.get_collider().is_in_group("transformation"):
+			c.get_collider().apply_central_impulse(-c.get_normal() * 100)
```
Comment (TrueFMartin) [inline scenes/characters/Player.gd:253]: Look at line 232 :)

**Review by TrueFMartin — COMMENTED** [review]
Can you explain what is supposed to be happening when you move the transformation around?

**Comment by malogal** [comment]
> Can you explain what is supposed to be happening when you move the transformation around?

When the transformation is pushed into the triglock, it will apply that transformation to the graph. success will be triggered when it is correctly modified. multiple transformations are possible this way.

**Comment by TrueFMartin** [comment]
> > Can you explain what is supposed to be happening when you move the transformation around?
> 
> When the transformation is pushed into the triglock, it will apply that transformation to the graph. success will be triggered when it is correctly modified. multiple transformations are possible this way.

But there is no graph and there is no visible area to push the transformation to (at least not yet) is that correct?

**Pull request merged**